source/sessions/driver-sessions.rst

```diff
@@ -991,95 +1014,102 @@ been upgraded to 3.6.
 Test Plan
 =========
 
-1. Pool is LIFO.
-    * This test applies to drivers with session pools.
-    * Call ``MongoClient.startSession`` twice to create two sessions, let us call them ``A`` and ``B``.
-    * Call ``A.endSession``, then ``B.endSession``.
-    * Call ``MongoClient.startSession``: the resulting session must have the same session ID as ``B``.
-    * Call ``MongoClient.startSession`` again: the resulting session must have the same session ID  as ``A``.
+1. Pool is LIFO
```

Comment:
Similar to my style comment above, I think it would be better to make these format changes in a new PR.